pkg/dsl/definition/template.go

```diff
@@ -361,11 +353,7 @@ func (td *traitDef) getTemplateContext(ctx process.Context, cli client.Reader, n
 		if err != nil {
 			return nil, err
 		}
-		if assist.IsOutputs {
-			outputs[assist.Name] = object
-		} else {
-			root[OutputFieldName] = object
-		}
+		outputs[assist.Name] = object
```

Comment:
I wonder whether we should check there are same assist.Name. If yes, errors should be returned.

Reply:
Yes, that should be checked in webhook, let me add a TODO here.